types/manager-types.yaml

```diff
@@ -636,6 +636,9 @@ node_types:
         description: Number of log files to keep when rotating.
         type: integer
         default: { get_input: rest_service_log_files_backup_count }
+      service_user:
```

Comment:
this should probably be consistent with other user/group properties... os_user, os_group etc...